lms/djangoapps/courseware/views/views.py

```diff
@@ -1328,12 +1335,12 @@ def financial_assistance_form(request):
             },
             {
                 'name': 'income',
-                'type': 'text',
+                'type': 'select',
                 'label': FA_INCOME_LABEL,
-                'placeholder': _('income in US Dollars ($)'),
+                'placeholder': '',
```

Comment:
Placeholder would be shown for the textfield as we have change it to dropdown placeholder won't be required.